hq/markdown/snyk.md

```diff
@@ -7,7 +7,11 @@ vulnerabilities in your application.
 
 ## Installing
 
-Snyk requires node, and `npm`.  To install on a mac, please use:
+Snyk requires node, and `npm`.
```

Comment:
Personal preference, however I prefer Yarn... (and have set up this project to use Yarn) Maybe:

> Snyk requires node, and a Node package manager such as `npm` or `Yarn`

I realise the build servers are currently using NPM, however I am hoping to remove this inconsistency

Reply:
Done.